src/Ganeti/HTools/ExtLoader.hs

```diff
@@ -122,8 +122,12 @@ loadExternalData opts = do
   now <- getClockTime
 
   let ignoreDynU = optIgnoreDynu opts
+      nodeMem = optStaticKvmNodeMemory opts
       eff_u = if ignoreDynU then [] else util_data
       ldresult = input_data >>= (if ignoreDynU then clearDynU else return)
+                            >>= (if nodeMem >= 0
```

Comment:
AFAIK this is a temporary workaround as this should already be handled in 2.17. Could you add a comment here reinforcing that?